config/mappings/assessment_mapping_fund_round.py

```diff
@@ -0,0 +1,47 @@
+from config.mappings.assessment_mapping import (
+    cof_r2w2_assessment_mapping as cof_r2w2,
+)
+from fsd_utils import CommonConfig
+
+fund_round_to_assessment_mapping = {
+    f"{CommonConfig.COF_FUND_ID}:{CommonConfig.COF_ROUND_2_ID}": cof_r2w2,
+}
+
+
+def transform_to_assessor_task_list_metadata(
+    fund_id: str, round_id: str
+) -> tuple[dict, dict]:
+    mapping = fund_round_to_assessment_mapping[f"{fund_id}:{round_id}"]
+
+    sections = [
+        {
+            "name": s["name"],
+            "sub_criterias": [
+                {"name": sc["name"], "id": sc["id"]}
+                for sc in s["sub_criteria"]
+            ],
+        }
+        for s in mapping["unscored_sections"]
+    ]
+
+    criterias = [
+        {
+            "name": c["name"],
+            "total_criteria_score": -1,  # TODO: Get dynamic state.
+            "total_criteria_score_possible": -1,  # TODO: ^^^
+            "weighting": c["weighting"],
+            "sub_criterias": [
+                {
+                    "name": sc["name"],
+                    "id": sc["id"],
+                    "score": -1,  # TODO: Get dynamic state.
+                    "theme_count": len(sc["themes"]),
+                    "status": "Not started",  # TODO: Get dynamic state.
+                }
+                for sc in c["sub_criteria"]
+            ],
+        }
+        for c in mapping["scored_criteria"]
+    ]
+
+    return sections, criterias
```

Comment:
Would be good to see some testing for this

Reply:
I've added https://github.com/communitiesuk/funding-service-design-assessment-store/pull/50/commits/f03237ce25be02fc3eadc29f7c9699c84b9310fa which sort of acts as a test/documentation for the eventually transformed structure (this is what is returned from the endpoint, the dataclass `AssessorTaskList` is built form, and used to dynamically populate the UI with config elements, and state (once part of the transformation)